cirq/ops/common_gates.py

```diff
@@ -1076,6 +1076,10 @@ class ISwapPowGate(eigen_gate.EigenGate,
 
     `cirq.ISWAP`, the swap gate that applies i to the |01> and |10> states,
     is an instance of this gate at exponent=1.
+
+    References:
+        "What is the matrix of the iSwap gate?"
+        https://quantumcomputing.stackexchange.com/questions/2594/
```

Comment:
I guess it sort of makes sense to reference this.  But I'm not sure it is really helpful, I mean does going to that question really help the user?

Reply:
We'll be sending a user who asks why we are doing this to that link, so we might as well have it here for users who have that question.

Reply:
We don't do this for other gates (similarly they could be defined differently, and indeed the handedness of single qubit rotations is done very much using both conventions) so it seems strange to call it out.

Reply:
Good point. I assume all other gates are not "defined differently", meaning deviating from standards texts.

Reply:
> handedness of single qubit rotations is done very much using both conventions

Didn't understand what this meant before i posted my comment. I agree.

Reply:
I think the fact that this question came up for this gate, and not for other gates, justifies the citation. I would be happy to add similar references to the other gates, e.g. to particular pages of mike and ike.